playwright/_impl/_helper.py

```diff
@@ -50,6 +50,7 @@
 RouteHandler = Union[Callable[["Route"], Any], Callable[["Route", "Request"], Any]]
 
 ColorScheme = Literal["dark", "light", "no-preference"]
+ReducedMotion = Literal["no-preference", "reduce"]
```

Comment:
```suggestion
ReducedMotion = Literal["reduce", "no-preference"]
```

Reply:
It causes linting error so let it be like the earlier one https://github.com/microsoft/playwright-python/pull/715/checks?check_run_id=2672816782